tooling/minitest/src/ub/compare_exchange.rs

```diff
@@ -22,9 +22,9 @@ fn compare_exchange_success() {
         print(load(local(1)), 3)
     );
 
-    // Doesn't if current doesn't match.
+    // Failure case: check that we do not perform a store
```

Comment:
The test doesn't actually print `local(0)` again, does it? So it doesn't check whether a store happened or not?

Reply:
I removed the wrong print.. Should I once again check both or is it enough to check that `local(0)` did not change?

Reply:
Seems reasonable to check both, why not.